charts/steward/README.md

```diff
@@ -109,7 +109,7 @@ Pipeline Run Controller:
 | <code>runController.<wbr/>tolerations</code> | (array of [`Toleration`][k8s-tolerations])<br/> The `tolerations` field of the Run Controller [pod spec][k8s-podspec]. | `[]` |
 | <code>runController.<wbr/>args.<wbr/>qps</code> | (integer)<br/> The maximum queries per second (QPS) from the controller to the cluster. | 5 |
 | <code>runController.<wbr/>args.<wbr/>burst</code> | (integer)<br/> The burst limit for throttle connections (maximum number of concurrent requests). | 10 |
-| <code>runController.<wbr/>args.<wbr/>logVerbosity</code> | (integer)<br/> The log verbosity. 0=error, 1=warning, 2=info, 3=extended, 4=debug, 5=trace (see also [logging-conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md#logging-conventions)) | 3 |
+| <code>runController.<wbr/>args.<wbr/>logVerbosity</code> | (integer)<br/> The log verbosity. 0=default, 1=info, 2=debug, 3=trace (see also [logging-conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md#logging-conventions)) | 1 |
```

Comment:
Do we agree on those log levels for our project? The linked document at least uses 1-5 with `trace` on `5`.
Yes, the numbers are meant relative and can be different for our project. But it could also mean that we cannot use a default verbosity to get all expected log entries, but need to set it different verbosity levels per module.

If the rest agrees, I would also be fine with it...

Reply:
Its fitting rather to the description of the logr and i suggested it kind of in my summery. But you are right, maybe we should keep suggested klog info levels...